src/core/componentSystems/TextSystem.js

```diff
@@ -87,6 +88,11 @@ export class TextSystem extends MRSystem {
             }
 
             this.updateStyle(entity);
+            if (entity instanceof Button) {
```

Comment:
just checking - is button the only current divEntity that needs a special case? or is it all div entities that can be within text

Reply:
only button.